examples/webgl_materials_normaldisplacementmap.html

```diff
@@ -185,6 +185,9 @@
 				var parameters = { fragmentShader: shader.fragmentShader, vertexShader: shader.vertexShader, uniforms: uniforms, lights: true, fog: false };
 				var material1 = new THREE.ShaderMaterial( parameters );
 
+				// Ensure that ENVMAP_MODE_REFLECTION is defined in shader
+				material1.envMap = reflectionCube;
```

Comment:
`reflectionCube` is already defined as a uniform. Don't add it again. The proper way to do this is to remove this change and add the following line as the first line of the fragment shader in `NormalDisplacementShader.js`:

		"#define ENVMAP_MODE_REFLECTION",